rites/roman1962/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@internal/rite-roman1962",
+  "version": "0.0.0",
+  "description": "Liturgical calendar for the Roman Rite according to the 1962 reform",
```

So, throwing a wrench, the calendar is technically the 1960 calendar, it was incorporated into the 1962 edition of the Roman Missal. so we probably need to refactor as `roman1960` (i know we don't have any calendars in here yet, but just hoping to keep it accurate)

```suggestion
  "description": "Liturgical calendar representing the General Roman Calendar of 1960",
```

---

Thank you for your comment and the historical reminder! However, the 1962 reform was much more significant (compared to the one in 1960), and it’s the one we use today as a reference when we talk about the "extraordinary form" of the Roman rite, or the "Tridentine Mass," or the "Traditional Mass." So, I’d be inclined to keep 1962 as the reference for that reason. What do you think?

---

i agree with the idea, but we are also calling the current calendar (in code) the `rite-roman1969`, which corresponds to the original promulgation of the calendar, not the updates (i.e. 2002, 2000, or 1975)...

I'm fine with your wording and i think it suffices to assuage my issues 👍

---

Thank you, and since these packages are not released for now, we still have time to test the naming.